packages/auth/amplify_auth_cognito/example/tool/DefineAuthChallengeLambda/src/index.js

```diff
@@ -0,0 +1,15 @@
+exports.handler = async (event) => {
+  if (event.request.session.length == 1 && event.request.session[0].challengeName == 'SRP_A') {
+    event.response.issueTokens = false;
+    event.response.failAuthentication = false;
+    event.response.challengeName = 'CUSTOM_CHALLENGE';
+  } else if (event.request.session.length == 2 && event.request.session[1].challengeName == 'CUSTOM_CHALLENGE' && event.request.session[1].challengeResult == true) {
+      event.response.issueTokens = true;
+      event.response.failAuthentication = false;
+      event.response.challengeName = 'CUSTOM_CHALLENGE';
```

Comment:
Challenge name should not be included when issuing tokens. It won't impact the front-end but is technically incorrect/not needed.